src/components/home-container/index.jsx

```diff
@@ -5,42 +5,51 @@ import Button from "components/base/button";
 import CreateTask from "components/task/create-new";
 import TaskCard from "components/task/existing-task/task-card";
 import FilterBtnContainer from "components/filter-btn-container";
+import NoTaskFound from "components/not-found/task";
 
 function HomeContainer() {
   const tasks = useSelector((state) => state.todo.tasks);
   const [isNewTaskRequested, setIsNewTaskRequested] = useState(false);
 
-  const handleSuccessfulTaskEntry = () => setIsNewTaskRequested(false);
-  const handleCreateBtnClick = () => setIsNewTaskRequested(true);
-
   return (
     <div className={`container mx-auto ${styles.homeWrapper}`}>
       <h1>Add Tasks</h1>
       <div className={`flex justify-between ${styles.buttonContainer}`}>
-        <Button onButtonClick={handleCreateBtnClick}>Create</Button>
+        <Button onButtonClick={() => setIsNewTaskRequested(true)}>
```

Comment:
Can we write a separate method and call that instead, inside the onButtonClick?

Reply:
According to the react [doc](https://react.dev/learn/responding-to-events), inline event handler is useful for shorter function. As the click event handler only updates the state & there is no chance of reusing this code so don.t you think we can write this updatestate inside inline event handler ? @fza-wtag vai
![Screenshot 2023-04-02 at 2 57 20 PM](https://user-images.githubusercontent.com/128278816/229342943-73dafb4b-b698-4a43-93c8-2844a53a13c8.png)

Reply:
Thanks for sharing. Resolving the conversation.